code/modules/mob/living/simple_animal/friendly/corgi.dm

```diff
@@ -16,66 +16,34 @@
 	response_harm   = "kicks"
 	see_in_dark = 5
 	mob_size = 8
+	max_nutrition = 250//Dogs are insatiable eating monsters. This scales with their mob size too
+	stomach_size_mult = 30
+	seek_speed = 6
 	possession_candidate = 1
 	holder_type = /obj/item/weapon/holder/corgi
 	var/obj/item/inventory_head
 	var/obj/item/inventory_back
 
+/mob/living/simple_animal/corgi/New()
+	..()
+	nutrition = max_nutrition * 0.3//Ian doesn't start with a full belly so will be hungry at roundstart
+	nutrition_step = mob_size * 0.15
+	nutrition_step = mob_size * 0.12
```

Comment:
mob sizes are different on eris, and really fucked up. these numbers may need rebalanced, better yet the mob sizes themselves should be changed